recipes/configs/gemma/2B_full.yaml

```diff
@@ -0,0 +1,83 @@
+# Config for multi-device full finetuning in full_finetune_distributed.py
+# using a gemma 2B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download --repo-id meta-llama/gemma-2b \
+#   --hf-token <HF_TOKEN> \
+#   --output-dir /tmp/gemma2
+#
+# To launch on 4 devices, run the following command from root:
+#   tune --nnodes 1 --nproc_per_node 4 full_finetune_distributed \
+#   --config gemma/2B_full \
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training
+# you can run:
+#   tune --nnodes 1 --nproc_per_node 4 full_finetune_distributed \
+#   --config gemma/2B_full \
+#   checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works best when the model is being fine-tuned on 2+ GPUs.
```

Comment:
nit: I think if we are running with `full_finetune_distributed` recipe it will *only* work on 2+ GPUs

Reply:
yeah, I update it in the latest version.